webhooks-extension/src/components/WebhookCreate/WebhookCreate.js

```diff
@@ -509,7 +513,7 @@ class WebhookCreatePage extends Component {
 
             <div className="row">
               <div className="help-icon" id="git-tooltip">
-                <TooltipIcon tooltipText="The URL of the git repository to create the webhook on.">
+                <TooltipIcon onClick={this.preventTooltipPost} tooltipText="The URL of the git repository to create the webhook on.">
```

Comment:
Instead of having to add this `onClick` to every Tooltip it would be better to define your own Tooltip component (wrapping the Carbon one) with a default onClick and use that. It will help to ensure consistency and make it easier to update all instances in future.

```js
const MyCustomTooltip = props => (
  <CarbonTooltip {...props} />
);

MyCustomTooltip.defaultProps = {
  onClick: e => e.preventDefault()
};

...

<MyCustomTooltip tooltipText="The URL of ..." />
```